Failure to: conda update anaconda

Current conda install:

               platform : win-64
          conda version : 4.3.21
       conda is private : False
      conda-env version : 4.3.21
    conda-build version : not installed
         python version : 3.6.1.final.0
       requests version : 2.12.4
       root environment : C:\Anaconda3  (writable)
    default environment : C:\Anaconda3
       envs directories : C:\Anaconda3\envs
                          C:\Users\tjmckinx\AppData\Local\conda\conda\envs
                          C:\Users\tjmckinx\.conda\envs
          package cache : C:\Anaconda3\pkgs
                          C:\Users\tjmckinx\AppData\Local\conda\conda\pkgs
           channel URLs : https://repo.continuum.io/pkgs/free/win-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/win-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/win-64
                          https://repo.continuum.io/pkgs/pro/noarch
                          https://repo.continuum.io/pkgs/msys2/win-64
                          https://repo.continuum.io/pkgs/msys2/noarch
            config file : C:\Users\tjmckinx\.condarc
             netrc file : None
           offline mode : False
             user-agent : conda/4.3.21 requests/2.12.4 CPython/3.6.1 Windows/8.1 Windows/6.3.9600
          administrator : False

`$ C:\Anaconda3\Scripts\conda-script.py update anaconda`




    Traceback (most recent call last):
      File "C:\Anaconda3\lib\site-packages\conda\exceptions.py", line 632, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "C:\Anaconda3\lib\site-packages\conda\cli\main.py", line 137, in _main
        exit_code = args.func(args, p)
      File "C:\Anaconda3\lib\site-packages\conda\cli\main_update.py", line 65, in execute
        install(args, parser, 'update')
      File "C:\Anaconda3\lib\site-packages\conda\cli\install.py", line 357, in install
        execute_actions(actions, index, verbose=not context.quiet)
      File "C:\Anaconda3\lib\site-packages\conda\plan.py", line 830, in execute_actions
        execute_instructions(plan, index, verbose)
      File "C:\Anaconda3\lib\site-packages\conda\instructions.py", line 247, in execute_instructions
        cmd(state, arg)
      File "C:\Anaconda3\lib\site-packages\conda\instructions.py", line 108, in UNLINKLINKTRANSACTION_CMD
        txn.execute()
      File "C:\Anaconda3\lib\site-packages\conda\core\link.py", line 263, in execute
        self.verify()
      File "C:\Anaconda3\lib\site-packages\conda\core\link.py", line 242, in verify
        self.prepare()
      File "C:\Anaconda3\lib\site-packages\conda\core\link.py", line 149, in prepare
        for pkg_info in self.packages_info_to_link)
      File "C:\Anaconda3\lib\site-packages\conda\core\link.py", line 149, in <genexpr>
        for pkg_info in self.packages_info_to_link)
      File "C:\Anaconda3\lib\site-packages\conda\core\link.py", line 54, in determine_link_type
        if hardlink_supported(source_test_file, target_prefix):
      File "C:\Anaconda3\lib\site-packages\conda\_vendor\auxlib\decorators.py", line 56, in _memoized_func
        result = func(*args, **kwargs)
      File "C:\Anaconda3\lib\site-packages\conda\gateways\disk\test.py", line 84, in hardlink_supported
        assert not lexists(test_file), test_file
    AssertionError: C:\Anaconda3\.tmp.index.json

--- a/conda/gateways/disk/test.py
+++ b/conda/gateways/disk/test.py
@@ -81,6 +81,8 @@ def hardlink_supported(source_file, dest_dir):
     test_file = join(dest_dir, '.tmp.' + basename(source_file))
     assert isfile(source_file), source_file
     assert isdir(dest_dir), dest_dir
+    if lexists(test_file):
+        rm_rf(test_file)
     assert not lexists(test_file), test_file
     try:
         create_link(source_file, test_file, LinkType.hardlink, force=True)

